prefect_twitter/media.py

```diff
@@ -0,0 +1,71 @@
+"""This is a module for interacting with Twitter media"""
+
+from functools import partial
+from pathlib import Path
+from typing import TYPE_CHECKING, Union
+
+from anyio import to_thread
+from prefect import get_run_logger, task
+
+if TYPE_CHECKING:
+    from io import IOBase
+
+    from prefect_twitter import TwitterCredentials
+
+
+@task
+async def media_upload(
+    filename: Union[Path, str],
+    twitter_credentials: "TwitterCredentials",
+    file: "IOBase" = None,
+    chunked: bool = None,
+    **kwargs: dict
+) -> int:
+    """
+    Use this to upload media to Twitter. Chunked media upload
+    is automatically used for videos.
+
+    Args:
+        filename: The filename of the image to upload.
+            This will automatically be opened unless file is specified.
+        twitter_credentials: Credentials to use for authentication with Tweepy.
+        file: A file object, which will be used instead of opening filename.
+            Note, filename is still required, for MIME type detection and
+            to use as a form field in the POST data.
+        chunked: Whether or not to use chunked media upload.
+            Videos use chunked upload regardless of this parameter.
+        kwargs: Additional keyword arguments to pass to media_upload.
```

Comment:
Can you link to some documentation so that users know what other kwargs are accpeted by `media_upload`?